index.md

```diff
@@ -17,52 +22,117 @@ Install [Atom](https://atom.io/). Atom is a program where we edit code - it is a
 
 ### 3) Install Xcode
 
-Download and install Xcode from the Apple App Store on your computer. This will probably take a little while to finish. Xcode is a huge suite of development tools published by Apple. If we wanted to develop software for the Apple Ecosystem (iPhone apps, macOS apps, etc), we would use Xcode as our editor. But even if we aren’t working in this ecosystem, Xcode provides some system dependencies that we’ll want to have available.
+[Xcode](https://developer.apple.com/xcode/) is a suite of development tools published by Apple. If we wanted to develop software for the Apple Ecosystem (iPhone apps, macOS apps, etc), we would use Xcode as our editor. But even though we're not building iPhone apps, Xcode provides some system dependencies that we need.
+
+Rather than download Xcode via the Apple Store, we can get a much smaller selection of necessary tools via our terminal. 
 
-Once the download is complete, open Xcode and agree to the SLA terms. Then, open the Terminal application, You can get to Terminal by pressing the `Command + Space` keys at the same time, then typing `Terminal` into the search. Once Terminal is open, type the following into the application: 
+Open the Terminal by pressing the `Command + Space` keys at the same time, which opens Spotlight, and then type `Terminal` into the search. Press the enter key, and type the following into the application, but without the `$`: 
 
 ```
-xcode-select --install
+$ xcode-select --install
 ```
 When prompted, enter your password. 
 
+You can verify that the installation was successful by typing `xcode-select` in your terminal. You should see something like this:
+
+```
+$ xcode-select
+xcode-select: error: no command option given
+Usage: xcode-select [options]
+
+Print or change the path to the active developer directory. This directory
+controls which tools are used for the Xcode command line tools (for example,
+xcodebuild) as well as the BSD development commands (such as cc and make).
+
+Options:
+  -h, --help                  print this help message and exit
+.
+.
+.
+```
+
+#### A note on convention
+
+when you see the `$` before a line, it means "enter what follows the dollar sign as a terminal command". Lines that don't have a `$` might be what is printed out as a _result_ of the command, or, if the line begins with `//`, it means it's a comment. A few `.` on lines means "omitted text here". For example:
```

Comment:
I love the context about the $ and what it's used for. 

I do worry that some folks may be overwhelmed by the context of the `//` (particularly since people likely don't know what a comment is in the context of programming) and `.`, but I may be totally wrong. 

We can add a question or questions about the clarity of the computer setup to see if folks are having any confusion in our technical pre-survey. If you want to help with those questions, let me know!

Reply:
This is a perceptive observation. I've updated this little section a bit to try to be a tad more clear; the real proof would be asking the students themselves if they got confused, in that survey.

If this survey is public, I can add this question myself. If not, would you be willing to add a question to this extent?